### PR TITLE
installation guide and footer added

### PR DESCRIPTION
**Pull request by abhishektanwar**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"liveServer.settings.port": 5501
+}
```
Comment (DivyanshSareen) [inline .vscode/settings.json:3]: add this to .gitignore

File: docsite.css [hunk 2]
```diff
@@ -90,6 +90,23 @@
...
+.footer-container i{
+	color: var(--toast-error-color);
+}
+
+.footer-container h3{
+	color: var(--white-color);
+}
+
+.margin-top-20{
+	margin-top: 20px;
```
Comment (DivyanshSareen) [inline docsite.css:107]: use relative units

**Review by DivyanshSareen — APPROVED** [review]
Good to merge 👍

**Review by humzahasan — APPROVED** [review]
UI looks good. 

Good to merge!

**Pull request merged**